winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -121,119 +110,12 @@ static void kerberos_ContextFree(KRB_CONTEXT* context)
 		context->target_name = NULL;
 	}
 
-	sspi_gss_release_cred(&minor_status, &context->cred);
-
-	if (context->gss_ctx)
-	{
-		sspi_gss_delete_sec_context(&minor_status, &context->gss_ctx, SSPI_GSS_C_NO_BUFFER);
-		context->gss_ctx = SSPI_GSS_C_NO_CONTEXT;
-	}
+	// if (context->gss_ctx)
```

Comment:
Should remove that before merge

Reply:
It should actually be uncommented; I'll do that